lib/google-apis/auth-client.js

```diff
@@ -92,35 +142,13 @@ RequestHandler.prototype.sendError = function(errorMessage, userCb) {
   var that = this;
   var cb = isFunction(userCb) ? userCb : function() {};
 
-  if (!that._shouldReportErrors) {
-
+  if (that._config.getShouldReportErrorsToAPI()) {
+    this._getProjectIdentifier(this._doRequest.bind(this, errorMessage, cb));
```

Comment:
Bound functions are significantly slower (until recently). `_doRequest` doesn't seem to have other uses. Is there a strong reason to change this from the original that was saving state through closure capture instead of `this` & bind?